model/src/main/kotlin/config/LicenseFilePatterns.kt

```diff
@@ -31,6 +31,11 @@ data class LicenseFilePatterns(
      */
     val licenseFilenames: Set<String>,
 
+    /**
+     * A set of globs that match typical notice filenames.
+     */
+    val noticeFilenames: Set<String>,
```

Comment:
This commit must be marked breaking, because `config.yml` will need to be modified in order to be de-serializable.

Reply:
I just noticed that as well, and was wondering: Why don't we assign default values to these properties to avoid this breakage?

Reply:
> Sure, I propose `empty` as default value. (As opposed to `LicenseFilePattern.DEFAUT.*`.)

Why? That does not seem to be very user friendly to me...

Reply:
Ok, I didn't provide a reason, but you also didn't. So, this is not helping to move the discussion forward.

Reply:
There was no immediate feedback from anyone in the community call to use custom `LicenseFilePatterns`. So I guess not having a default value for the new `noticeFilenames` property is okay, as most user implicitly use `LicenseFilePatterns.DEFAULT` anyway.